recipes/example/meta.yaml

```diff
@@ -79,4 +79,7 @@ extra:
     # Always check with the people listed below if they are OK becoming maintainers of the recipe. (There will be spam!)
     - LisaSimpson
     - LandoCalrissian
+<<<<<<< HEAD
```

Comment:
@rluria14 if you check the github interface you should be able to see these mistakes here.

This PR should only contain you on meta.yaml file.